tools/ci_connector_ops/ci_connector_ops/utils.py

```diff
@@ -130,3 +134,48 @@ def get_acceptance_test_config(connector_name: str) -> Tuple[str, Dict]:
     except FileNotFoundError:
         logging.warning(f"No {ACCEPTANCE_TEST_CONFIG_FILE_NAME} file found for {connector_name}")
         return None, None
+
+@dataclass
+class Connector:
+    # TODO a couple of functions defined at the root level of this module can be moved into this class.
```

Comment:
Why is this a todo? just from the comment seems simple enough to do now 😄

Reply:
As this will change other module I didn't want to clutter the PR with unrelated changes. But I will before it becomes a tech debt.